<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphView</class>
 <widget class="QWidget" name="GraphView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>668</width>
        <height>423</height>
       </rect>
      </property>
      <widget class="QCustomPlot" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>431</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QPushButton" name="ButtonCreateGraph">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>150</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: green;</string>
       </property>
       <property name="text">
        <string>Create graph</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>10</y>
         <width>217</width>
         <height>120</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ymax</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineYmin">
          <property name="text">
           <string>-100</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineYmax">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineXmax">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineXmin">
          <property name="text">
           <string>-100</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ymin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Xmax</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Xmin</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../../graph/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
